src/main/java/edu/ucsb/cs156/happiercows/repositories/CowDeathRepository.java

```diff
@@ -1,7 +1,14 @@
 package edu.ucsb.cs156.happiercows.repositories;
-import edu.ucsb.cs156.happiercows.entities.CowDeath;
+
+import java.util.Optional;
+
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import edu.ucsb.cs156.happiercows.entities.CowDeath;
 
 public interface CowDeathRepository extends CrudRepository <CowDeath, Long>{
-    
+	Optional<CowDeath> getCowsKilledByCommonsIdAndUserId(Long commons_id, Long user_id );
```

Comment:
I don't know why there is an extra tab being added. I deleted the tab and pushed again but it is still here.